autogluon/inference/docker/0.7/py3/Dockerfile.cpu

```diff
@@ -10,12 +10,10 @@ LABEL com.amazonaws.sagemaker.capabilities.accept-bind-to-port=true
 LABEL com.amazonaws.sagemaker.capabilities.multi-models=true
 
 LABEL maintainer="Amazon AI"
-LABEL dlc_major_version="1"
+LABEL dlc_major_version="2"
 
 # PYTORCH_SKIP_CUDNN_COMPATIBILITY_CHECK - see https://github.com/autogluon/autogluon/issues/2534
-ENV MXNET_CUDNN_LIB_CHECKING=0 \
-    MXNET_CUDNN_AUTOTUNE_DEFAULT=0 \
-    PYTORCH_SKIP_CUDNN_COMPATIBILITY_CHECK=1
+ENV PYTORCH_SKIP_CUDNN_COMPATIBILITY_CHECK=1
```

Comment:
Do we still need this with torch 11.3 and CUDA 11.7 can you verify?

Reply:
just double-checked looks like it's working now; removed; closed [#2534](https://github.com/autogluon/autogluon/issues/2534)

Reply:
removed; confirmed 1.13.1 resolved [#2534](https://github.com/autogluon/autogluon/issues/2534)